app/views/datasets/_describe_extra_fields_edit.html.erb

```diff
@@ -1,21 +1,23 @@
+<% isDigital = false %>
 <% if !@files.nil? && !@files.empty? %>
   <%= f.hidden_field :medium, :value=>Sufia.config.data_medium["Digital"] %>
+  <% isDigital = true %>
 <% else %>
-
   <% if @dataset.medium[0] == Sufia.config.data_medium["Digital"] %>
     <% showDigital = "inline" %>
   <% else %>
     <% showDigital = "none" %>
-  <% end %>
+<% end %>
+ 
+<% if !isDigital %>
   <div class="fieldset">
-    <p class="small">Is your data in digital format?</p>
+      <p class="small reqlabel">Is your data in digital format?</p>   
     <fieldset class="small-text">
       <label panel="dataset-format-digital">
-        <%= f.radio_button :medium, Sufia.config.data_medium["Digital"], :checked => (@dataset.medium[0] == Sufia.config.data_medium["Digital"]), :onchange => "toggleDigitalFieldsDisplay('digital')", :class => "medium-digital", data: {"category" => "documentation"} %><span></span> Yes
+        <%= f.radio_button :medium, Sufia.config.data_medium["Digital"], :checked => isDigital, :class => "required", :onchange => "toggleDigitalFieldsDisplay('digital')", :class => "medium-digital required", data: {"category" => "documentation"} %><span></span> Yes        
```

Comment:
This field is never going to be checked as this bit of the form only displays if isDigital is false. You need to display checked = true | false based on the field value not based on if there are files or not